packages/athena/sodasql/dialects/athena_dialect.py

```diff
@@ -56,6 +56,14 @@ def default_connection_properties(self, params: dict):
             'catalog': 'AwsDataCatalog'
         }
 
+    def safe_connection_data(self):
+        return [
```

Comment:
is there a concept of host or something in athena? if so I think that is what we want to push into `datasource_id` for this warehouse.

Reply:
Do you know what information we can use to uniquely identify an account or host @vijaykiran

Reply:
indeed, Athena is challenging to create a unique-enough hash without sensitive info

Reply:
do you have a sample of what it looks like so I can see?

Reply:
That looks very good to me!